ci_framework/roles/reproducer/templates/play.yml.j2

```diff
@@ -1,11 +1,90 @@
 ---
+{% if use_content_provider is defined and use_content_provider | bool -%}
+- name: Configure CRC for insecure registry
+  hosts: crcs
```

Comment:
Not 100% related to this PR, but we are starting using groups and I'm not sure we are documenting it

Reply:
soooo that group is 100% managed by the reproducer. Still, good point regarding the group in zuul job definition.